.github/.test/samples.json

```diff
@@ -800,12 +800,6 @@
         "Client: Python"
       ]
     },
-    {
-      "input": "python-server-flask-petstore-python2.sh",
-      "matches": [
-        "Server: Python"
-      ]
-    },
```

Comment:
looks like a rebase/merge was done incorrectly to include these changes that are already present in the master

Reply:
Yes I used a rebase/merge ... bad!
What's the best way to solve this issue?

Reply:
one way is to create a new PR and cherry-pick commits.

Reply:
Done! - Please have a look to [the new PR for jaxrs jersey3-server](https://github.com/OpenAPITools/openapi-generator/pull/16335)